runtimevar/drivertest/drivertest.go

```diff
@@ -26,10 +26,17 @@ import (
 	"github.com/google/go-cmp/cmp"
 )
 
+const (
+	noWait = 1 * time.Microsecond
```

Comment:
These names are confusing, why `noWait` is 1us while `defaultWait` is 0? Sounds like `noWait` should be 0?

Reply:
Good point. I renamed `noWait` to `shortWait` and added comments.

defaultWait is 0, which causes driver implementations to use their default (usually 30 seconds).